lib/elixir/src/elixir_import.erl

```diff
@@ -65,7 +65,8 @@ record_warn(Meta, Ref, Opts, Added, E) ->
 calculate(Meta, Key, Opts, Old, File, Existing) ->
   New = case keyfind(only, Opts) of
     {only, Only} when is_list(Only) ->
-      ok = ensure_keyword_list(Meta, File, Only, only),
```

Comment:
I removed this match for symmetry with `ensure_no_duplicates` and `ensure_no_special_form_conflict`; and because it will raise a `CompileError` if the given value is not a keyword list 🙂